globocom/directional-navigation
# Add cypress support for tests

## Changes
- Fix an exception on method call at render of the page.
- Add cypress to run tests against the library
## How to test
- Remove the check before call and reload the example. It will throw an exception
- run `npm run cypress` to execute the test selection or to execute headless with `npm run cypressheadless`
